server/integration-tests/data-survives-restart.sh

```diff
@@ -0,0 +1,47 @@
+#/bin/bash
+
+if [ ! -f "stack.yaml" ]; then
+    echo "Run test in server package directory"
+    exit 1
+fi
+
+# PERARATIONS
+stack build
+DATA_FILE=`mktemp`
+echo '{}' > $DATA_FILE
```

Comment:
Might want to quote this, since it is also quoted elsewhere:

```bash
echo '{}' > "$DATA_FILE"
```

Reply:
Done